### ISSUE TITLE
Input lag when '━' character is visible

### ISSUE THREAD

**Issue by adombeck** [Guake/guake#2229]
I'm experiencing a weird issue where Guake is becoming incredibly slow at processing my keyboard inputs after I used `git diff` or `git show` with https://github.com/dandavison/delta as the pager.

I tracked the issue down to the '━' character which is printed as separators by delta.

I can reliably reproduce the issue by running this a few times in Guake:

```bash
echo ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
```

The more often I run it, the slower Guake becomes.

When I clear the terminal with `clear`, there is no input lag anymore.

GNOME Terminal is not affected, so I assume it's a Guake issue.

It seems to be unrelated to the shell, at least I can reproduce it both with zsh as bash.

<details><summary>$ guake --support</summary>

Guake Version:		3.10

Vte Version:		0.75.92

Vte Runtime Version:	0.75.92

--------------------------------------------------
GTK+ Version:		3.24.41

GDK Backend:		<GdkX11.X11Display

--------------------------------------------------
Desktop Session: gnome

--------------------------------------------------
Display: :0

RGBA visual: True

Composited: True

* Monitor: 0 - HDMI-1
    * Geometry:		2560 x 1440 at 0, 0
    * Size:		700 x 390 mm²
    * Primary:		True
    * Refresh rate:	59.912 Hz
    * Subpixel layout:	unknown

**Comment by egmontkob** [comment]
Thanks for the report! VTE co-developer here. I can reproduce the issue with gnome-terminal, too.

I've forwarded the report upstream:
https://gitlab.gnome.org/GNOME/vte/-/issues/2757
https://gitlab.gnome.org/GNOME/vte/-/issues/2758

**Comment by adombeck** [comment]
> I can reproduce the issue with gnome-terminal, too.
> 
> I've forwarded the report upstream

Thanks! I wasn't able to reproduce the issue in GNOME Terminal in my initial tests, where I printed the '━' about 500 times which already slowed down Guake a lot but did not have any noticeable effect in GNOME Terminal. Only after printing the character about 20000 times now I begin to see some slow down in GNOME Terminal as well, so maybe you're right and the bug is actually in VTE and Guake is just more strongly affected by the bug than GNOME Terminal for some reason.

**Comment by egmontkob** [comment]
Out of curiosity: do you have gnome-terminal 3.51-ish (gtk3) or 3.97 (gtk4)?

**Comment by adombeck** [comment]
```
$ gnome-terminal --version
# GNOME Terminal 3.51.90 using VTE 0.75.92 +BIDI +GNUTLS +ICU +SYSTEMD
```

**Comment by egmontkob** [comment]
Fixed for the forthcoming stable vte 0.76 release.

**Comment by adombeck** [comment]
Awesome! I just briefly scrolled through the two upstream issues you created and can tell you put a lot of effort into this, so thanks a lot for all you work, it's really appreciated! :)
